pymc3/sampling.py

```diff
@@ -159,11 +181,17 @@ def sample(draws=500, step=None, init='auto', n_init=200000, start=None,
         up the corresponding backend (with "mcmc" used as the base
         name).
     chain : int
-        Chain number used to store sample in backend. If `njobs` is
+        Chain number used to store sample in backend. If `chains` is
         greater than one, chain numbers will start here.
+    chains : int
```

Comment:
It might be a bit confusing with both `chain` and `chains` in the keyword...

Reply:
I agree. I don't really see a good reason for `chain` to exist in the first place, but I'm not sure we can delete it safely. Any ideas?